### PR TITLE
Camtrawl2

### PR DESCRIPTION
**Pull request by Erotemic**
This branch has a working camtrawl pipeline. No segfaults. Works with opencv 2.4 and 3.x as well as python 2.7 and 3.x. (Note opencv 3.x results are a bit better).

### CODE REVIEW

File: plugins/camtrawl/matlab/Processing_with_species_id.m [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+%                                project_points2.m 
+%                                   rodrigues.m
+%                                   rigid_motion.m
+%                                   normalize_pixel.m
+%                                       comp_distortion_oulu.m
+%                                       comp_distortion.m
+
+clear all; clc; close all;
+
+%% DEFINE PARAMETERS
```
Comment (linus-sherrill) [inline plugins/camtrawl/matlab/Processing_with_species_id.m]: Unexpected line termination. Other places in this file too.

Comment (Erotemic) [reply plugins/camtrawl/matlab/Processing_with_species_id.m]: Seems to be in the matlab files. At this point these are only here for historical reasons @mattdawkins, can these be deleted?

Comment (Erotemic) [reply plugins/camtrawl/matlab/Processing_with_species_id.m]: @linus-sherrill The newlines should now be normalized.

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Compiled Object files
+ Compiled Object files
```
Comment (linus-sherrill) [inline .gitignore]: Will removing the leading '#' cause any problems?

Comment (Erotemic) [reply .gitignore]: Not sure, but removing it was an accident so I've undone it.

File: examples/measurement_using_stereo/README.rst [hunk 3]
```diff
@@ -17,3 +17,112 @@ examples covering fish measurement using stereo.
 `Coming Soon`_  on Pull Request 25
```
[the remark this answers is not in the document]
Comment (Erotemic) [reply examples/measurement_using_stereo/README.rst]: done

File: CMakeLists.txt [hunk 4]
```diff
@@ -8,6 +8,13 @@ project( VIAME )
 
 include( CMakeDependentOption )
 
+
+# If VIAME_FORCEBUILD is False, we will assume we dont need to rebuild any
+# other superbuild proejct (except for viame). This makes development builds
+# much quicker.
+option(VIAME_FORCEBUILD True)
+mark_as_advanced( VIAME_ENABLE_KWIVER )
```
[the remark this answers is not in the document]
Comment (Erotemic) [reply CMakeLists.txt]: done

**Pull request merged**